### ISSUE TITLE
Is there any button to pause the simulation or rendering in GUI?

### ISSUE THREAD

**Issue by zichunxx** [bulletphysics/bullet3#4549]
Hi!

I'm new to `pybullet` and was wondering if is there any button to pause the simulation or rendering in GUI, like the `space bar` for `Mujoco`. 

I want to observe some details when necessary during the simulation. Is it possible? 

Thanks in advance!

**Comment by justincredible** [comment]
I don't think this is possible when running the demos as standalone programs, but if you run the ExampleBrowser pressing 'i' will pause/resume stepping through the simulation.

I would recommend searching through https://pybullet.org/Bullet/phpBB3/ for these sorts of questions.
